package.json

```diff
@@ -19,7 +19,7 @@
     "test:slim": "ava",
     "test:report": "nyc --skip-full --reporter=lcov ava",
     "commit": "git cz",
-    "copy-static": "cpy \"server/**/*\" \"!**/*.{j,t}s\" \"migrations\" \"config/yml/*.yml\" dist/ --parents"
+    "copy-static": "cpy \"server/**/*\" \"!**/*.{j,t}s\" \"migrations\" \"conf.d\" dist/ --parents"
```

Comment:
IMO config shouldn’t be copied into dist/ tree, the config loading should read from repo root / directory above dist instead by default